regtests/t_spark_sql/src/spark_sql_s3_cross_region.sh

```diff
@@ -42,9 +42,6 @@ curl -i -X PUT -H "Authorization: Bearer ${SPARK_BEARER_TOKEN}" -H 'Accept: appl
   http://${POLARIS_HOST:-localhost}:8181/api/management/v1/principal-roles/service_admin/catalog-roles/spark_sql_s3_cross_region_catalog \
   -d '{"name": "catalog_admin"}' > /dev/stderr
 
-curl -H "Authorization: Bearer ${SPARK_BEARER_TOKEN}" -H 'Accept: application/json' -H 'Content-Type: application/json' \
-  "http://${POLARIS_HOST:-localhost}:8181/api/catalog/v1/config?warehouse=spark_sql_s3_cross_region_catalog"
-echo
```

Comment:
It looks like this is being removed because the output will change depending on `AWS_REGION_FOR_CROSS_REGION_TEST`, is that right?

Rather than removing it, I wonder if we could do some `jq` magic to make it env-variable-agnostic. 

Anyway, it's not an issue to just remove it imo.

Reply:
Yeah, it would be a useful feature to add, but for right now, i just want to make the tests pass :)